src/components/CodeBox.vue

```diff
@@ -45,32 +46,28 @@ function clickedButton(word: IWord) {
     currentEditingBug.value = new Bug(word.id, ErrorType.UNDEFINED, wordString);
     showModal.value = true;
   } else {
-    removeBugCode(word.id);
+    removeBugWord(word.id);
   }
 }
 
-function submitBug() {
-  selectedBugs.value.push(currentEditingBug.value);
+function submitBug(bug: IBug) {
+  selectedBugs.value.push(bug);
 }
 
-function hiddenModal() {
-  console.log('HIDE MODAL');
-}
-
-function removeBugCode(wordId: number) {
+function removeBugWord(wordId: number) {
   selectedBugs.value = selectedBugs.value.filter((bug) => bug.wordId != wordId);
 }
 
-function isWordSelectedBug(wordId: number) {
+function isSelectedBugWord(wordId: number) {
   return selectedBugs.value.find((bug) => bug.wordId == wordId) != null;
 }
 
-function isWordSpace(wordId: number): boolean {
+function isSpaceWord(wordId: number): boolean {
```

Comment:
```suggestion
function isWordSpace(wordId: number): boolean {
```

Reply:
I see you like my change :joy: 

I'm torn about which is better.
I mostly changed it to unify it with `BugWord` methods but hey don't exist anymore so I will roll it back